plaso/cli/views.py

```diff
@@ -300,28 +304,56 @@ def Write(self, output_writer):
 class MarkdownTableView(BaseTableView):
   """Markdown table view."""
 
-  def Write(self, output_writer):
-    """Writes the table to the output writer.
+  def _WriteHeader(self, output_writer):
+    """Writes a header.
 
     Args:
       output_writer (OutputWriter): output writer.
     """
     if self._title:
-      output_writer.Write('### {0:s}\n\n'.format(self._title))
+      output_writer.Write('{0:s} {1:s}\n\n'.format(
+          '#' * self._title_level, self._title))
 
-    if not self._columns:
-      self._columns = ['' for _ in range(0, self._number_of_columns)]
+    if self._columns:
+      output_writer.Write(' | '.join(self._columns))
+      output_writer.Write('\n')
 
-    output_writer.Write(' | '.join(self._columns))
-    output_writer.Write('\n')
+      output_writer.Write(' | '.join(['---' for _ in self._columns]))
+      output_writer.Write('\n')
+    else:
+      output_writer.Write('<table>\n')
 
-    output_writer.Write(' | '.join(['---' for _ in self._columns]))
-    output_writer.Write('\n')
+  def _WriteRow(self, output_writer, values):
+    """Writes a row of values aligned to the column width.
 
-    for values in self._rows:
-      values = ['{0!s}'.format(value) for value in values]
-      output_writer.Write(' | '.join(values))
+    Args:
+      output_writer (OutputWriter): output writer.
+      values (list[object]): values.
+    """
+    if self._columns:
+      row_values = ['{0!s}'.format(value) for value in values]
+      output_writer.Write(' | '.join(row_values))
       output_writer.Write('\n')
+    else:
+      row_values = ''.join([
+         '<td>{0!s}</td>'.format(value) for value in values[1:]])
+      output_writer.Write((
+          '<tr><th nowrap style="text-align:left;vertical-align:top">{0!s}</th>'
```

Comment:
I'm surprised to see HTML in MarkdownTableView(), is this WAI?

Reply:
Yes apparently this is how to do a vertical table in markdown